elliottlib/cli/find_bugs_sweep_cli.py

```diff
@@ -314,14 +312,19 @@ def categorize_bugs_by_type(bugs: List[Bug], advisory_id_map: Dict[str, int], ma
     not_found = set(tracker_bugs) - found
     if not_found:
         not_found_with_component = [(b.id, b.whiteboard_component) for b in not_found]
-        red_prefix("Tracker Bugs Warning: ")
-        click.echo("The following (tracker bug, package) were found BUT not attached,"
-                   " since no corresponding brew build was found attached to any advisory. "
-                   "First attach builds to the correct advisory and rerun to attach the bugs, "
-                   "or exclude the bug ids in the assembly definition")
-        click.echo(not_found_with_component)
-        raise ValueError(f'No builds found for CVE (bug, package): {not_found_with_component}. Either attach '
-                         f'builds or exclude the bugs in the assembly definition')
+        # Do not error out if a single advisory is requested to be operated on
+        if not default_advisory_type:
+            red_prefix("Tracker Bugs Warning: ")
+            click.echo("The following (tracker bug, package) were found BUT not attached,"
```

Comment:
Voting for logging rather than `click.echo`. If only because then we dont need to mock the method in unittests.